src/chapters/web-apis/studio.rst

```diff
@@ -0,0 +1,411 @@
+.. index:: ! headless API
+
+Studio: Consuming the Coding Events API With Postman
+====================================================
+
+The UI of a browser is designed to make simple ``GET`` requests for URLs entered into its address bar. This design works great for browsing sites, but 
+falls short when working with APIs. Anything beyond a ``GET`` request is difficult to send via a browser address bar alone. Think about what is needed to 
+create a new CodingEvent. This type of request contains a body. Our MVC application included a view to allow us to test inputs. Our API, however, is 
+**headless**. It does not contain the client-side form. In order to test how it handles requests then, we need a way to interact with the API server without 
+the browser. In this studio, we work with Postman to explore how APIs can be consumed.
+
+Setup
+-----
+
+Install Postman
+^^^^^^^^^^^^^^^
+
+If you haven't done so already, :ref:`install Postman <postman-installation>`.
+
+Fork and Clone the API Source Code
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+With Postman installed, we're ready to fork and clone the `CodingEvent API <https://github.com/LaunchCodeEducation/coding-events-api/tree/1-sqlite>`__. 
+Although it is implemented differently from CodingEvents MVC, you will find that most of the features from the MVC application have been supported through 
+endpoints in the API.
+
+.. admonition:: Note
+
+   Our focus in this course is on operations and as such we will not be going into the development of the API. However, feel free to explore the source 
+   code if you are curious about the similarities and differences between the .NET MVC and API implementations.
+
+Let's begin by forking and cloning the repo onto our machine. In your Powershell or terminal window, move into a directory were you plan to save your local
+copy of the API codebase.
+
+.. sourcecode:: bash
+
+   > git clone https://github.com/<GitUsername>/coding-events-api
+
+For this studio, we want to have the ``1-sqlite`` checked out. This branch has an API with a single (``CodingEvent``) 
+resource and a built-in SQLite database. 
+
+.. admonition:: Note
+
+	Including a SQLite database in this project means you don't need to have your MySQL server running to test the API. We won't get into what this looks like
+	and instead just concentrate on testing the API endpoints.
```

Comment:
Kind of ambiguous. "We won't get into what this looks like ..." sounds like it is in reference to not having a MySQL server running. Maybe that is what it is about, but on the first read, I thought it was actually about SQLite.